app/src/main/java/org/projectbuendia/client/sync/ChartDataHelper.java

```diff
@@ -211,32 +211,37 @@ public Chart getChart(String uuid) {
                 String label = Utils.getString(c, ChartItems.LABEL, "");
                 if (parentRowid == null) {
                     // Add a section.
-                    switch (Utils.getString(c, ChartItems.SECTION_TYPE)) {
-                        case "TILE_ROW":
-                            ChartSection tileGroup = new ChartSection(label);
-                            tileGroups.add(tileGroup);
-                            tileGroupsById.put(rowid, tileGroup);
-                            break;
-                        case "GRID_SECTION":
-                            ChartSection rowGroup = new ChartSection(label);
-                            rowGroups.add(rowGroup);
-                            rowGroupsById.put(rowid, rowGroup);
-                            break;
+                    String SectionType = Utils.getString(c, ChartItems.SECTION_TYPE);
+                    if (SectionType != null) {
```

Comment:
You are right. Sometimes I wonder where was my mind when I do somethings like this...